docs/developer-docs/latest/developer-resources/database-apis-reference/graphql-api.md

```diff
@@ -13,6 +13,10 @@ To use the GraphQL API, install the [GraphQL](/developer-docs/latest/plugins/gra
 
 The GraphQL API allows performing queries and mutations to interact with the [content-types](/developer-docs/latest/development/backend-customization/models.md#content-types) through Strapi's [GraphQL plugin](/developer-docs/latest/plugins/graphql.md). Results can be [filtered](#filters), [sorted](#sorting) and [paginated](#pagination).
 
+::: note
+Media Library [folders](/user-docs/latest/media-library/organizing-assets-with-folders.md) are not accessible through the GraphQL API.
+:::
+
```

Comment:
Isn't it weird to mention that? It gives the idea that it should be there. But it's far from sure that it should. I would not mention that here, neither in rest-api. However I would see it in the ML plugin part. "Folders are an admin feature and are not part or the Rest nor the GraphQL API"

Reply:
I get your point, but I'm unsure about this one. 🤔 I mean:
- we can rephrase it as you suggest, 
- but I'm pretty sure we will soon have users start asking how to query folders, and the best place to warn them about this is in the APIs documentation themselves.

Also asking @derrickmehaffy because they would have to handle users' requests :) What do you think, Derrick?

Reply:
I'd leave it out too I think - I'd like to see feature requests coming in for that, if our users think it should be there. The callout gives the impression as if this won't be possible, but I'm pretty sure this will be in one of the next iterations.

Reply:
I didn't try to upload because I'm lazy :p but the code is there yes : https://github.com/strapi/strapi/blob/features/ML-folder/packages/core/upload/server/controllers/content-api.js#L96

Reply:
Regarding mentioning it, I guess at some points it will be available but I'm not sure what it will look like. Maybe we want to separate the 2 use cases and use different folders for the uploaded files. It may not be as simple as giving access to the current folders.

Reply:
The correct Name is "API Uploads" : https://github.com/strapi/strapi/blob/features/ML-folder/packages/core/upload/server/constants.js#L16

Reply:
Following our discussions (here and on Slack), I've deleted the callouts in the REST and GraphQL APIs documentation and introduced another callout in the Upload (dev) docs. Thank you!